doc/helpers.js

```diff
@@ -0,0 +1,7 @@
+exports.isAsync = function(){
```

Comment:
It would be nice to have this file formatted and following our style conventions